dashboard/config/initializers/atomic_save.rb

```diff
@@ -0,0 +1,10 @@
+# Provides `#atomic_save!` which skips the default transaction
+# wrapping behavior of `#save!`.
+#
+# Only use in cases where validations/callbacks will never
+# need to trigger any rollback of the operation.
+module ActiveRecord
+  module Persistence
+    alias_method :atomic_save!, :save!
```

Comment:
This seems a little scary.  Is skipping validation a common pattern for speeding up DB writes in Rails?

How does this change the behavior?  It looks like we're just creating an `atomic_save!` alias of `save!`?